Documentation/Upgrade/Backup/Index.rst

```diff
@@ -19,3 +19,5 @@ Also you may prefer to upgrade a copy of your site first, if there have been a
 lot of changes and some of them might interfere with functions of your site.
 See the `ChangeLog <https://docs.typo3.org/typo3cms/extensions/core/>`_ to
 check that.
+
+For :ref:`more detailed information about TYPO3 backups see the TYPO3 Explained manual<t3coreapi:security-backups>`.
```

Comment:
I think there is a space missing here between anchor text and < and I would not put so much text in anchor text. I will push another commit.

Reply:
fyi, the space was not really necessary - it rendered correctly. But we usually have it, so it looked "weird" for me. In any case, I pushed the commit but can be removed if you do not agree.

Reply:
space or no-space - either is fine with me.